packages/experience/src/pages/ForgotPassword/ForgotPasswordForm/index.tsx

```diff
@@ -76,10 +77,19 @@ const ForgotPasswordForm = ({
         // Cache or update the forgot password identifier input value
         setForgotPasswordIdentifierInputValue({ type, value });
 
+        // Set current identifier input value for code verification
+        setIdentifierInputValue({ type, value });
```

Comment:
Is this expected?

Reply:
Sorry, it's a test code in the git stash, and committed by mistake. Removed.